### DISCUSSION TITLE
Feature request: insert end marker when there are blank lines

### DISCUSSION THREAD

**Discussion by esamson** [scalameta/scalafmt#4030] [Ideas]
The Scala 3 reference [suggests](https://docs.scala-lang.org/scala3/reference/other-new-features/indentation.html#when-to-use-end-markers-1):

> ... An end marker makes sense if
>
> * the construct contains blank lines, or

Is there a way to enforce this using scalafmt?

**Comment by kitbellew** [comment]
not this condition

**Reply by esamson** [reply]
Could it be a feature request? I feel like it makes for better readability at a glance.
